test/data/mock-state.json

```diff
@@ -120,7 +120,8 @@
     "networkConfigurations": {
       "testNetworkConfigurationId": {
         "rpcUrl": "https://testrpc.com",
-        "chainId": "0x1"
+        "chainId": "0x1",
+        "nickname": "Custom Mainnet RPC"
```

Comment:
Non-Blocking change, but Why are we adding a nickname here?

Reply:
Because tests bomb on `name={network.nickname}` without it.